CodeMaid.config

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This project uses .editorconfig. This CodeMaid.config file should not be committed since it is your personal preferences. Please remove and add to .gitignore